cypress/integration/visual-test/combobox-test.spec.js

```diff
@@ -0,0 +1,30 @@
+import { baseURL, checkEyes } from '../../util';
+describe('Visual Regression Testing', () => {
+  before(() => {
+    cy.eyesOpen({
+      appName: 'Clarity',
+      testName: 'combobox',
+      browser: { width: 1024, height: 768, name: 'chrome' },
+    });
+  });
+  after(() => {
+    cy.eyesClose();
+  });
+  it('check basic combobox', () => {
+    cy.visit(baseURL + 'combobox/basic');
+    cy.get('.clr-combobox .clr-combobox-trigger').click();
+    checkEyes('basic combobox');
+  });
+
+  it('check optional-menu combobox', () => {
+    cy.visit(baseURL + 'combobox/optional-menu');
+    cy.get('.clr-combobox:nth-child(2) .clr-combobox-trigger').click();
```

Comment:
These demos change sometimes, and `nth-child(#)` becomes too brittle over time. It would be better to add new `id` attributes to these elements in the demos in case they are moved around.

Reply:
done